pyaedt/emit_core/results/revision.py

```diff
@@ -181,6 +181,28 @@ def is_domain_valid(self, domain):
         engine = self.emit_project._emit_api.get_engine()
         return engine.is_domain_valid(domain)
 
+    @pyaedt_function_handler()
+    def get_instance_count(self, domain):
+        """
+                Return the number of instances in the domain for the current revision.
+
+                Parameters
+                ----------
+                domain :
+                    ``InteractionDomain`` object for constraining the analysis parameters.
+        Returns
+        --------
+        count : float
```

Comment:
Maybe it could just be an `int` here.
```suggestion
        count : int
```